tests/covalent_dispatcher_tests/_core/data_modules/job_manager_test.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+from functools import partial
+
+import pytest
+
+from covalent._workflow.transport import _TransportGraph
+from covalent_dispatcher._core.data_modules.job_manager import (
+    get_jobs_metadata,
+    set_cancel_requested,
+    set_cancel_result,
+    set_job_handle,
+)
+
+
+def get_mock_tg():
+    tg = _TransportGraph()
+    tg.add_node_by_id(0, job_id=1)
```

Comment:
it is not necessary to create a new `TransportGraph` method just for this test. `TransportGraph.add_node()` should suffice.

Reply:
For more context, `TransportGraph.add_node_by_id()` was initially added to support cancelling postprocessing -- but that is no longer necessary in view of upcoming changes to how postprocessing will handled.

Reply:
@cjao ah I see... alright will deprecate this method and update the affected tests